test/circuit/library/ansatzes/test_ucc.py

```diff
@@ -19,14 +19,15 @@
 from qiskit_nature import QiskitNatureError
 from qiskit_nature.circuit.library import UCC
 from qiskit_nature.converters.second_quantization import QubitConverter
-from qiskit_nature.mappers.second_quantization import JordanWignerMapper, ParityMapper
+from qiskit_nature.mappers.second_quantization import JordanWignerMapper
 from qiskit_nature.operators.second_quantization import FermionicOp
 
 
 def assert_ucc_like_ansatz(test_case, ansatz, num_spin_orbitals, expected_ops):
     """Assertion utility."""
-    excitation_ops = ansatz.excitation_ops()
 
+    # excitation_ops = ansatz.excitation_ops()[::2] if include_imaginary else ansatz.excitation_ops()
```

Comment:
Is the code that was commented here not needed anymore? If yes, the comment ought to be removed